apps/sms/views/inbox/js/inbox.js

```diff
@@ -991,9 +1004,12 @@ var InboxView = {
     clearTimeout(this.timeouts.onDraftSaved);
     this.timeouts.onDraftSaved = null;
 
-    this.timeouts.onDraftSaved = setTimeout(function hideDraftSavedBanner() {
-      this.draftSavedBanner.classList.add('hide');
-    }.bind(this), this.DRAFT_SAVED_DURATION);
+    this.timeouts.onDraftSaved = setTimeout(
+      () => this.draftSavedBanner.classList.add('hide'),
+      this.DRAFT_SAVED_DURATION
+    );
+
+    this.notifyAboutSavedDraft = false;
```

Comment:
nit: I'd set this to false in beforeEnter, so that the use and the reset are close from each other.